Formula/bitrise.rb

```diff
@@ -12,17 +12,6 @@ class Bitrise < Formula
   end
 
   depends_on "go" => :build
-  depends_on "godep" => :build
```

Comment:
`vendor`d in the repo https://github.com/bitrise-io/bitrise

`stepman` and `envman` are downloaded on the fly, when required, into `$HOME/.bitrise/tools`